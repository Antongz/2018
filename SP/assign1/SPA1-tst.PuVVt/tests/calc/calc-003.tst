3 dup * print dup * print dup * print
