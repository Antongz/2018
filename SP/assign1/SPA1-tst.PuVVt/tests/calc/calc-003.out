9
81
6561
